### DISCUSSION TITLE
RENEW CANVA PRO

### DISCUSSION THREAD

**Discussion by AnaACC** [community/community#28188] [GitHub Education] [answered]
Hello there!

My Canva Pro stopped working by the GitHub Student Package, I guess my time limit has expired. Is there a way to renew Canva Pro by using this package? Can somebody help me, please?

Tnxxx

**Comment by atuschek0** [comment, marked as answer]
Unfortunately I just looked on the GitHub Student Developer Pack site [here](https://education.github.com/pack/join) and I'm not seeing Canva Pro here anymore. This is odd because I saw it on there not too long ago, it may not be offered anymore 😭

**Reply by bassa846** [reply]
Thank you for contacting GitHub Education about the Canva benefit. The Canva benefit is no longer offered in the GitHub Student Developer Pack.

For students who are already on the 12 month benefit, you will continue to have access for the remainder of your 12 month offer period & can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.

**Reply by Tiana80** [reply]
aweee mannn 💔💔💔

**Reply by aravinth1998** [reply]
Could you kindly provide more details on the process for renewing the pack?

**Comment by Abhinay-art** [comment]
I can't find canva pro in github student developer pack.
I think github stopped providing canva pro so can anyone please help me to get canva pro.

**Reply by bassa846** [reply]
@Abhinay-art Please take a look at the response I provided above to another user. 

Thank you for contacting GitHub Education about the Canva benefit. The Canva benefit is no longer offered in the GitHub Student Developer Pack.

For students who are already on the 12 month benefit, you will continue to have access for the remainder of your 12 month offer period & can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.

**Reply by DebugAgrawal** [reply]
I'm already on 12 month benefit, but unable to access canva pro
